src/woo_search/search_index/client.py

```diff
@@ -204,7 +191,7 @@ def get_search_results(
                 "omschrijving^1.2",
                 "attachment.content",
             ],
-            fuzziness=2,  # distance (1 is typically okay for typo's, two is more fuzzy)
```

Comment:
there's no such top-level key for `simple_query_string`

Reply:
We can put `~2` at the end of the provided query. That would also add fuzzieness just how we provide the boost by `^2`

Reply:
I don't think we can blindly append that to the query input, we would have to check each token separately.

Let's see whether we really need it, since by default transposition is already applied which should cover typos to some extent

Reply:
I also don't think we need it, but in case we do I thought it was nice to have it documented somewhere.